src/main/java/com/boot/amazon/controller/ProductController.java

```diff
@@ -12,14 +12,15 @@
 public class ProductController {
     private final ProductSevice productSevice;
     private final ProductMapper productMapper;
+    private static final int PRODUCTS_AMOUNT = 1000;
 
     public ProductController(ProductSevice productSevice, ProductMapper productMapper) {
         this.productSevice = productSevice;
         this.productMapper = productMapper;
     }
 
     @GetMapping("/top-thousand-most-commented")
-    public ProductGetRequestDto getThousandTopCommentedItems() {
-        return productMapper.map(productSevice.getTopThousand());
+    public ProductGetRequestDto getTopCommentedItems() {
```

Comment:
Point here was not to make it constant but to use @RequestParam with default value option. So user will be able to pass custom amount, but if he doesn't do that we use default value

Reply:
Ups :) Fixed that